### PR TITLE
[new release] carton, carton-lwt and carton-git (0.2.0)

### PR DESCRIPTION
**Pull request by dinosaure**
Implementation of PACKv2 file in OCaml

- Project page: <a href="https://github.com/mirage/ocaml-git">https://github.com/mirage/ocaml-git</a>
- Documentation: <a href="https://mirage.github.io/ocaml-git/">https://mirage.github.io/ocaml-git/</a>

### CODE REVIEW

File: packages/git/git.3.1.0/opam [hunk 1]
```diff
@@ -30,9 +30,9 @@ depends: [
   "mimic"
   "cstruct" {>= "5.0.0"}
   "angstrom" {>= "0.14.0"}
-  "carton"
-  "carton-lwt"
-  "carton-git"
+  "carton" {= "0.1.0"}
```
Comment (mseri) [inline packages/git/git.3.1.0/opam:33]: What about using
```suggestion
  "carton" {< "0.2.0"}
```
? There is no point in allowing for a potential bugfix backport in this case?

File: packages/git/git.3.1.1/opam [hunk 2]
```diff
@@ -30,9 +30,9 @@ depends: [
   "mimic"
   "cstruct" {>= "5.0.0"}
   "angstrom" {>= "0.14.0"}
-  "carton"
-  "carton-lwt"
-  "carton-git"
+  "carton" {= "0.1.0"}
```
Comment (mseri) [inline packages/git/git.3.1.1/opam:33]: Same question here

File: packages/git/git.3.1.0/opam (hunk 1 above)
Comment (dinosaure) [reply packages/git/git.3.1.0/opam]: You are right, :+1: fixed!

**Comment by mseri** [comment]
Thanks

**Pull request merged**